src/store/annotations/__tests__/selectors-test.ts

```diff
@@ -1,9 +1,21 @@
 import annotationsState from '../__mocks__/annotationsState';
-import { getActiveAnnotationId, getAnnotation, getAnnotations, getAnnotationsForLocation } from '../selectors';
+import {
+    getIsAnnotationsInitialized,
```

Comment:
`getIsInitialized`? It's already in the `annotations` selector module, so the name itself doesn't need to be namespaced again.